### DISCUSSION TITLE
Scoping on list of named selection

### DISCUSSION THREAD

**Discussion by Indrajeetrt** [ansys/pydpf-core#1813] [Q&A]
Hello there, Good day!!! I wanted to check is there any feasibility to scope on the list of named selection directly ? here is my below code where "ns" is list of named selections
"ns_op = dpf.operators.scoping.on_named_selection()
ns_op.inputs.data_sources.connect(data_source)
ns_op.inputs.requested_location.connect('Nodal')
ns_op.inputs.named_selection_name.connect(ns)"

or any possibilities to get results based on named selection for list of time scoping ?

**Comment by PProfizi** [comment]
Hello @Indrajeetrt,
Sadly the only way right now to provide a named selection to a result operator is as a `Scoping`.

The code you provided above is thus the best way to get a scoping corresponding to a named selection, to then feed it in other operators.

This is however a potential feature request we could investigate.
